Makefile.include

```diff
@@ -363,6 +363,12 @@ include $(RIOTBASE)/Makefile.features
 include $(RIOTMAKE)/pseudomodules.inc.mk
 include $(RIOTMAKE)/defaultmodules.inc.mk
 
+# handle removal of default modules
+USEMODULE += $(filter-out $(DISABLE_MODULE), $(DEFAULT_MODULE))
```

Comment:
I'm wondering whether this line should go be moved to `dependency_resolution.inc.mk`